### DISCUSSION TITLE
Tangem not supported in WalletConnect V2

### DISCUSSION THREAD

**Discussion by johncastorina** [WalletConnect/walletconnect-monorepo#2620] [Developer Support]
The forced update to Wallet Connect V2 is coming with several problems. 

My application has several users that all rely on Tangem wallet, which as of now only supports V1 of WalletConnect as a way to connect to dApps.

I have had to upgrade my dApp to use Wallet Connect V2, and now many of my users are locked out.

What is the timeline for getting Tangem support built into V2? Is this something the Wallet Connect team does, or is this up to Tangem to build support.

**Comment by finessevanes** [comment]
>What is the timeline for getting Tangem support built into V2? Is this something the Wallet Connect team does, or is this up to Tangem to build support.

This is something Tangem wallet needs to implement.

**Reply by spencer-tb** [reply]
After contacting Tangem it looks like they have updated to WalletConnect V2.

I think its the other services that use WalletConnect but have not upgraded from V1 to V2. 

For example, I was having issues with MyCrypto when trying to sign a message using WalletConnect. MyCrypto have not updated it on their end but Tangem have. Signing a message on Etherscan worked fine for me: https://etherscan.io/verifiedSignatures

**Reply by gittyup-ship-it** [reply]
hi, i'm using safe multisig and tangem and having problems.  is it because safe hasn't migrated to v2? i have tokens stuck in safe wallet that need to be migrated within 30 days otherwise i lose them : (

**Reply by gittyup-ship-it** [reply]
hi spencer, how do i sign using etherscan?

**Comment by gittyup-ship-it** [comment]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/143585022/28cd0680-b90a-4a67-8efe-94709a803fe9)

**Comment by gittyup-ship-it** [comment]
it's not tangems issue 
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/143585022/679ef2ec-b901-442f-a1f1-6ace294afbb3)
